### ISSUE TITLE
http-server-middleware has been deprecated

### ISSUE THREAD

**Issue by rohitagre** [tuupola/slim-jwt-auth#86]
Hi, 
switching from `http-interop/http-server-middleware` to `psr/http-server-middleware`
please see https://github.com/http-interop/http-server-middleware

Can you suggest any local changes to be done just in the mean time to keep things running?

I get the following error
```Fatal error: Declaration of Tuupola\Middleware\JwtAuthentication::process(Psr\Http\Message\ServerRequestInterface $request, Interop\Http\Server\RequestHandlerInterface $handler): 
Psr\Http\Message\ResponseInterface must be compatible with Psr\Http\Server\MiddlewareInterface::process(Psr\Http\Message\ServerRequestInterface $request, Psr\Http\Server\RequestHandlerInterface $handler): 
Psr\Http\Message\ResponseInterface in /Users/jammer/Code/localpull/rm/api/vendor/tuupola/slim-jwt-auth/src/JwtAuthentication.php on line 31

**Comment by tuupola** [comment]
The package `http-interop/http-server-middleware` was broken for a while. They should have fixed the problem now.

http-interop/http-server-middleware#6
http-interop/http-server-handler#13.

Will release version with `psr/http-server-middleware` most likely tomorrow.

**Comment by tuupola** [comment]
Problem should now also be also fixed in `3.0.0-rc.4`.
